boards/avr-rss2/include/rss2.h

```diff
@@ -0,0 +1,39 @@
+
```

Comment:
Please move these defines with proper doxygen documentation in board.h. Then remove rss2.h.